melange/init

```diff
@@ -0,0 +1,76 @@
+#!/bin/busybox sh
+set -e
+set -x
+
+/bin/mount -t proc proc -o nodev,nosuid,hidepid=2 /proc
+/bin/mount -t devtmpfs -o nosuid,noexec devtmpfs /dev
+/bin/mount -t sysfs sys -o nodev,nosuid,noexec /sys
+/bin/mount -t tmpfs -o nodev,nosuid,noexec tmpfs /tmp
+
+# Setup tty/pty
+/bin/mkdir /dev/pts
+/bin/mount -t devpts devpts -o noexec,nosuid,newinstance,ptmxmode=0666,mode=0620,gid=tty /dev/pts/
+/bin/mount --bind /dev/pts/ptmx /dev/ptmx
+
+# ldconfig is run to prime ld.so.cache for glibc packages which require it.
+if [ -x /sbin/ldconfig ]; then
+	/sbin/ldconfig /lib /usr/lib /usr/lib64 || :
```

Comment:
Why `|| :` ? do you have a case where this fails and thats ok?

As it is, you're just swallowing an error that is going to pop up later and be harder to debug later.

I wonder why we should have to run /sbin/ldconfig. Shouldn't our built images work optimally on read-only filesystem?

Reply:
Thanks, fixed this, I think it was a leftover from older debugging